### ISSUE TITLE
Faster sampling of marathon model

### ISSUE THREAD

**Issue by jhrcook** [fonnesbeck/gp_regression#3]
I found that changing the `marathon_model` sampling initializer to `"advi+adapt_diag"` fit the model far faster. ADVI took ~12,000 steps to converge and then NUTS took about 3 minutes (my computer is not very fast, so I suspect you and most others would see far faster times). I doubt this is new information for you, but I figured it would be useful since this model is probably the slowest fitting one in the notebooks (which is why you didn't fit it live in your lecture at PyData 2019).

**Comment by fonnesbeck** [comment]
Yeah, initialization is an art. Thanks for pointing this out, and I will certainly update prior to giving the course again.
